pages.es/common/tgpt.md

```diff
@@ -0,0 +1,37 @@
+# tgpt
+
+> Habla con un chatbot de IA sin necesidad de claves API.
+> Proveedores disponibles: `openai`, `opengpts`, `koboldai`, `phind`, `llama2`, `blackboxai`.
+> Más información: <https://github.com/aandrew-me/tgpt>.
+
+- Chatea con el proveedor por defecto (GPT-3.5-turbo):
+
+`tgpt "{{prompt}}"`
+
+- Inicia el modo interactivo [m]ultilínea:
+
+`tgpt --multiline`
+
+- Genera [i]mágenes y las guarda en el directorio actual:
+
+`tgpt --image "{{prompt}}"`
+
+- Genera [c]ódigo con el proveedor por defecto (GPT-3.5-turbo):
+
+`tgpt --code "{{prompt}}"`
+
+- Chatea con un proveedor específico [q]uietamente (sin animaciones):
+
+`tgpt --provider {{openai|opengpts|koboldai|phind|llama2|blackboxai}}} --quiet --whole "{{prompt}}"`
+
+- Genera y ejecuta comandos [s]hell utilizando un proveedor específico (con un prompt de confirmación):
```

Comment:
```suggestion
- Genera y ejecuta comandos de intérprete utilizando un proveedor específico (con confirmación):
```

With `prompt de confirmación` I understand something like `A question? [Y/N]`, is this what you meant @kant?

Reply:
Yes!